### DISCUSSION TITLE
Base NLP Model

### DISCUSSION THREAD

**Discussion by apurvgarg04** [armenak/DataDefender#178] [Q&A] [answered]
Do we have the code for the NLP model along with the trained models?

**Comment by armenak** [comment, marked as answer]
You will see all available info [here](https://opennlp.apache.org/).

**Reply by apurvgarg04** [reply]
Do we have the data that the model was trained on? Wanted to understand the model, and improve the model without the adapter based approach.

**Reply by armenak** [reply]
No, we do not. All we have is the information provided by OpenNLP.
